display/d.mon/list.c

```diff
@@ -52,7 +52,8 @@ void list_mon(char ***list, int *n)
     while ((dp = readdir(dirp)) != NULL) {
 	int ret;
 
-        if (!dp->d_name || dp->d_name[0] == '.')
+        /* 'dp->d_name' will always be true... */
+        if (!*dp->d_name || dp->d_name[0] == '.')
```

Comment:
```suggestion
        if (!dp->d_name[0] || dp->d_name[0] == '.')
```
Just for clarification in the code that d_name is an array, not a pointer, see `man 3 readdir`.

Reply:
Yes, `*db->d_name` will only work for instantiated strings. My bad. Updated as suggested.